resources/js/components/EnrolledPractical.vue

```diff
@@ -0,0 +1,382 @@
+<template>
```

Comment:
@softdiddy whats this file for? I thought that was the page that Said worked on last? See this https://github.com/cs-dept-ibbul/vlab/pull/27 and please delete this file from your PR.